src/services/findAllReferences.ts

```diff
@@ -1366,6 +1382,17 @@ namespace ts.FindAllReferences.Core {
         }
     }
 
+    function hasOwnConstructor(classDeclaration: ClassLikeDeclaration): boolean {
+        return !!getClassConstructorSymbol(classDeclaration.symbol);
+    }
+
+    function findInheritedConstructorReferences(classDeclaration: ClassLikeDeclaration, state: State): void {
+        if (hasOwnConstructor(classDeclaration)) return undefined;

```

Comment:
Don't you need to look if this class is extending another class and look for its constructor even though this doesn't have constructor?

Reply:
Yes. The reasoning is that this function will only be called when we are looking for a class' (`A`) constructor and we find a reference that is like `class B extends A`. So I think it is enough to check if `B` doesn't have is own constructor declared.